### ISSUE TITLE
Could you please provide the full version code ?

### ISSUE THREAD

**Issue by MsterDC** [mil-ad/prospr#1]
hi,
this is excellent work, but when I tried to reproduce the results in the paper I noticed that this is not the full version of the code as the "struct trimming" doesn't seem to be included, can you provide the full code? I will be very grateful

**Comment by panshoudeng** [comment]
Yes, I have the same need and would appreciate it if you could provide the complete code.
